torch/quantization/fx/qconfig_utils.py

```diff
@@ -13,6 +13,22 @@
 QConfigAny = Union[torch.quantization.QConfig,
                    torch.quantization.QConfigDynamic, None]
 
+def add_device_to_obs_ctr_in_qconfig(qconfig_dict, device='cpu'):
```

Comment:
do we want to modify qconfig_dict? I feel it would be better to keep it since it's user input?

Also I feel it might be better to provide this argument when we actually insert observers instead of changing the configs, wondering why this does not work? https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/prepare.py#L219

Reply:
The issue is that for weights and other observers that don't actually get inserted, they exist only as a constructor until they are instantiated and used. (https://fburl.com/diffusion/1k1fdoea). So the options are to edit the constructor or edit the from_float function for every module. There's probably a way to edit only the constructor that gets propagated to the modules, but when I tried that (https://fburl.com/diffusion/mt1l207v) , it seems to insert the constructor at multiple different points and it overwrites it.

Reply:
I'm going to remove these changes, it seems like there's significant disagreement about the desired direction.

Reply:
edit: changes were made such that the original qconfig is unaffected